src/main/java/com/teragrep/net_01/channel/context/IngressImpl.java

```diff
@@ -250,4 +259,46 @@ private void activateBuffers(List<BufferLease> bufferLeases) {
     public AtomicBoolean needWrite() {
         return needWrite;
     }
+
+    @Override
+    public void register(Clock clock) {
+        lock.lock();
+        try {
+            if (!interestedClocks.isEmpty()) {
+                throw new IllegalStateException("only one interested clock is allowed");
```

Comment:
Maybe change the formatting to `Unable to register ingress clock, only one interested clock is allowed`